src/index.js

```diff
@@ -20,6 +20,21 @@ export function isResponseCached(requestKey) {
   return Boolean(responseCache[requestKey]);
 }
 
+export function getCachedResponse(requestKey) {
+  return responseCache[requestKey] || null;
+}
+
+
+export function writeToCache(requestKey, res) {
+  if (isResponseCached(requestKey)) {
+    responseCache[requestKey] = res;
+    return responseCache[requestKey];
+  }
+  throw new CacheMissError(
```

Comment:
same here, decided to throw since i don't think one should be allowed to write something to the cache that isn't already there

Reply:
hmm, I don't know it was my initial idea, but I think i agree with you, especially if i was using this offline.